launcher/sdw_updater_gui/UpdaterAppUiQt5.py

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
```

Comment:
For packaging, this file is handled by https://github.com/freedomofpress/securedrop-workstation/blob/main/rpm-build/SPECS/securedrop-workstation-dom0-config.spec#L82-L83